README.md

```diff
@@ -8,18 +8,35 @@ Installation
 ============
 
 describe how to install renpy and/or any other tools we may use in the future.
+Download Ren'Py version 7.4.7 from https://www.renpy.org/release/7.4.7 for your opperating system. (Game is being developed for Windwos first due to it's popularity, it will be ported to other Operating Systems later.)
+
+Download the WhimsySimp Game Repository from https://github.com/DokiDokiDead/WhimsySimp.git and add it to your Ren'Py installation folder(Example: D:\Program Files\Ren'Py\renpy-7.4.7-sdk ). 
+
+For Windows: Open Ren'Py via the renpy.exe file inside your installation folder. There will be a menu labled "Projects", in which there will be three game projects available: "Tutorial" "The Question" and "WhimsySimp-Master", assuming all you have done is add this game to a fresh install.
+Click on "WhimsySimp-Master" and start fiddling about with your branch of the game!
 
 Wiki/Documentation
 ==================
 
-decide to add documentation in a google drive or make a wiki and use here.
+Ren'Py release used for development: https://www.renpy.org/release/7.4.7
+QuickStart guide for Ren'Py: https://www.renpy.org/doc/html/quickstart.html
+Ren'Py Reference Manual: https://www.renpy.org/doc/html/
 
-I personally would like to keep documentation internally since we'd likely have to link to other groups's work such as their art folder and such. Much easier imo to link around in google docs.
+For access to internal developer notes and discord, message one of the project managers.
```

Comment:
Capitalize Discord in this line

Reply:
Sentence should probably include 'the' before 'Discord' but that is an issue for the next PR for the README.